.gitignore

```diff
@@ -8,6 +8,8 @@
 
 # Unignore all dirs
 !*/
+# But ignore build
+build/
```

Comment:
Could you please explain why we need that?
By default, git won't add new directories to commit.
Plus, the build directory might have any name.

Reply:
Sorry, it was temporary solution for me and it shouldn't have been commited. I have already fixed this.

Reply:
I see. What is the problem? Maybe I can suggest more accurate fix.

Reply:
I have placed build directory inside my local repository. Because it needn't to be commited, I wanted to hide it from modified files. 
Now I moved build directory out of repository.